app/src/main/java/com/github/multimatum_team/multimatum/viewmodel/DeadlineListViewModel.kt

```diff
@@ -59,6 +60,14 @@ class DeadlineListViewModel @Inject constructor(
     fun getDeadlines(): LiveData<Map<DeadlineID, Deadline>> =
         _deadlines
 
+    /**
+     * get a deadline from the repository.
+     */
+    fun getDeadline(id: DeadlineID): Deadline =
+        _deadlines.value!![id]!!
```

Comment:
Sanity check, but there seems to be a lot of `!!` here. Are you sure that nothing can be null here?